genai-perf/tests/test_common.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+from unittest.mock import MagicMock, patch
+
+from genai_perf.subcommand.common import run_perf_analyzer
+
+
+class TestCommon:
+    @patch("genai_perf.subcommand.common.subprocess.run")
```

Comment:
Per Harshini's request, moved these into a test_common.py folder because run_perf_analyzer is a shared function in common.py.